### PR TITLE
tools.collect_libs now use self.cpp_info.libdirs

### LINKED ISSUE #836: Document tools.collect_libs() search in self.cpp_info.libdirs
Document PR conan-io/conan#3503

### PR DESCRIPTION
**Pull request by danimtb**
closes #836

### CODE REVIEW

File: reference/tools.rst [hunk 1]
```diff
@@ -1151,29 +1151,31 @@ tools.collect_libs()
...
     def package_info(self):
+        self.cpp_info.libdirs = ["lib", "other_libdir"]  # Deafult value is 'lib'
         self.cpp_info.libs = tools.collect_libs(self)
 
 For UNIX libraries staring with **lib**, like *libmath.a*, this tool will collect the library name **math**.
 
 **Parameters:**
-    - **conanfile** (Required): A `ConanFile` object from which to get the `package_folder`.
-    - **folder** (Optional, Defaulted to ``"lib"``): The subfolder where the library files are.
+    - **conanfile** (Required): A `ConanFile` object to get the `package_folder` and ``cpp_info``.
```
Comment (jgsogo) [inline reference/tools.rst]: `ConanFile` and `package_folder` should go with double quotes, shouldn't they?

**Comment by jgsogo** [comment]
This PR is the documentation for this one (https://github.com/conan-io/conan/pull/3503), right?

**Comment by danimtb** [comment]
Yes, it is linked in the issue indicated in the description.

**Comment by danimtb** [comment]
Change-log not needed anymore. And code PR was already merged.

**Pull request merged**